src/main/resources/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -0,0 +1,278 @@
+openapi: 3.0.0
+info:
+  title: Education and Work Plan API
+  version: '1.0.0'
+  description: Education and Work Plan API
+  contact:
+    name: Matt Wills
+    email: matt.wills@digital.justice.gov.uk
+servers:
+  - url: 'http://localhost:3000'
+#
+# --------------------------------------------------------------------------------
+#
+paths:
+  #
+  # --------------------------------------------------------------------------------
+  # RESTful style endpoints start here
+  # --------------------------------------------------------------------------------
+  #
+
+  #
+  # Goals
+  # --------------------------------------------------------------------------------
+  '/prisoners/{prisonNumber}/action-plan':
+    parameters:
+      - name: prisonNumber
+        description: The ID of the Prisoner (confusingly not the prison) who may have created (or is in the process of creating) an "Action Plan", consisting of one or more Goals.
```

Comment:
I'd be tempting to remove the "(confusingly not the prison)" - its only confusing to newbies to HMPPS. I suspect other devs who might read/use this spec know this 😁

Reply:
Funnily enough, I had the same thought and nearly removed it :)